docs/process/pull-requests.md

```diff
@@ -5,10 +5,10 @@ a consistent process for contributions from the core team and the community.
 
 ## The Review Process
 
-1. **Contributor** opens pull request.
+1. **Contributor** opens a pull request.
 1. When it's ready for review, add a comment to the pull request.
```

Comment:
should we suggest the use of draft PRs here instead?

Reply:
I’d say the usage of drafts in this repo is only really for PRs in early stages and maybe major changes.

Reply:
@rafeca yeah i think encouraging that would be great! i try to use them any time i'm working on something until its ready for review.

Reply:
Thanks for the feedback! I've changed the text slightly to encourage using draft PRs when they're not ready for review.